recipes/awslambdaric/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "awslambdaric" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awslambdaric-{{ version }}.tar.gz
+    sha256: dad646f566aa7ec9b7179f16ca6741a2bea148abec6ed5947f86d00607e0a9a2
+  # Please see awslabs/aws-lambda-cpp#94, remove once aws-lambda-cpp is upgraded to 0.2.7
+    patches:
+      - relax-simplejson.patch
+      - aws-lambda-cpp-0.2.6-update-backward-cpp.patch
+  - url: https://raw.githubusercontent.com/bombela/backward-cpp/v1.6/backward.hpp
+    sha256: 404aa26ecc36be08f92c03b05505561e064f6f1f7cffcbda77a7d4959e4c07f2
+    folder: deps/patches
+    fn: backward.h
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [linux]
+    - {{ compiler('cxx') }}  # [linux]
+    - automake  # [linux]
+    - autoconf  # [linux]
+    - cmake  # [linux]
+    - libtool  # [linux]
+    - make  # [linux]
+    - pkg-config  # [linux]
+  host:
+    - pip
+    - python
+  run:
+    - importlib-metadata >=2.0,<3  # [py<38]
+    - python
+    - simplejson >=3.17.2,<3.18
+
+test:
+  imports:
+    - awslambdaric
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/aws/aws-lambda-python-runtime-interface-client
+  summary: AWS Lambda Runtime Interface Client for Python
+  license: BSD-3-Clause AND Apache-2.0
+  license_file:
+    - NOTICE
+    - LICENSE
```

We need to include the licenses for the third-party-libraries that it vendors (static curl and aws-lambda-runtime-cpp)

```suggestion
    - LICENSE
    - THIRD-PARTY-LICENSES
```